src/Ryujinx.Audio/Renderer/Server/Splitter/SplitterDestinationVersion1.cs

```diff
@@ -103,7 +104,8 @@ public void Update<T>(in T parameter) where T : ISplitterDestinationInParameter
 
                 parameter.MixBufferVolume.CopyTo(MixBufferVolume);
 
-                if (!IsUsed && parameter.IsUsed)
+                bool resetPrevVoume = isPrevVolumeResetSupported ? parameter.ResetPrevVolume : !IsUsed && parameter.IsUsed;
```

Comment:
```suggestion
                bool resetPrevVolume = isPrevVolumeResetSupported ? parameter.ResetPrevVolume : !IsUsed && parameter.IsUsed;
```
Need to change the use below, too.